File: students/students/src/MON.SPA/src/models/printTemplateModel.js
export class PrintTemplateModel {
  constructor(obj = {}) {
    this.id = obj.id || null;
    this.basicDocumentId = obj.basicDocumentId;
    this.basicDocumentName = obj.basicDocumentName;
    this.printFormId = obj.printFormId,
    this.printFormEdition = obj.printFormEdition,
    this.name = obj.name || null;
    this.description = obj.description || null;
    this.institutionId = obj.institutionId || null;
    this.hasContents = obj.hasContents || false;
    this.left1Margin = obj.left1Margin || 0;
    this.top1Margin = obj.top1Margin || 0;
    this.left2Margin = obj.left2Margin || 0;
    this.top2Margin = obj.top2Margin || 0;
  }
}

File: students/students/src/MON.Models/Institution/PrintTemplate/PrintTemplateModel.cs
namespace MON.Models.Institution.PrintTemplate
{
    public class PrintTemplateModel
    {
        public int? Id { get; set; }
        public int? InstitutionId { get; set; }
        public int? RuoRegId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int BasicDocumentId { get; set; }
        public int Left1Margin { get; set; }
        public int Top1Margin { get; set; }
        public int Left2Margin { get; set; }
        public int Top2Margin { get; set; }
        public int PrintFormId { get; set; }
    }
}
